OpenRA.Mods.Common/Traits/Render/WithCrateBody.cs

```diff
@@ -20,8 +20,6 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Renders crates with both water and land variants.")]
 	class WithCrateBodyInfo : ITraitInfo, Requires<RenderSpritesInfo>, IQuantizeBodyOrientationInfo, IRenderActorPreviewSpritesInfo
 	{
-		public readonly string[] Images = { "crate" };
```

Comment:
This breaks the ability to have randomized crate variants, if we still want that.